src/main/resources/licenses/CC-BY-4.0.txt

```diff
@@ -0,0 +1,245 @@
+                  technical modifications necessary to exercise the Licensed Rights,
+                  including technical modifications necessary to circumvent Effective
+                  Technological Measures. For purposes of this Public License, simply
+                  making modifications authorized by this Section 2(a)(4) never produces Adapted Material.
+          5.      Downstream recipients.
+                      A. Offer from the Licensor – Licensed Material.
+                         Every recipient of the Licensed Material automatically receives an
+                         offer from the Licensor to exercise the Licensed Rights under the terms
+                         and conditions of this Public License.
+                      B. No downstream restrictions.
+                         You may not offer or impose any additional or different terms or
+                         conditions on, or apply any Effective Technological Measures to, the
+                         Licensed Material if doing so restricts exercise of the Licensed Rights
+                         by any recipient of the Licensed Material.
+          6.      No endorsement.
+                  Nothing in this Public License constitutes or may be construed as
+                  permission to assert or imply that You are, or that Your use of the
+                  Licensed Material is, connected with, or sponsored, endorsed, or granted
+                  official status by, the Licensor or others designated to receive
+                  attribution as provided in Section 3(a)(1)(A)(i).
+
+    b.    Other rights.
+
+          1.     Moral rights, such as the right of integrity, are not
+                 licensed under this Public License, nor are publicity, privacy, and/or
+                 other similar personality rights; however, to the extent possible, the
+                 Licensor waives and/or agrees not to assert any such rights held by the
+                 Licensor to the limited extent necessary to allow You to exercise the
+                 Licensed Rights, but not otherwise.
+          2.     Patent and trademark rights are not licensed under this Public License.
+          3.     To the extent possible, the Licensor waives any right to
+                 collect royalties from You for the exercise of the Licensed Rights,
+                 whether directly or through a collecting society under any voluntary or
+                 waivable statutory or compulsory licensing scheme. In all other cases
+                 the Licensor expressly reserves any right to collect such royalties.
+
+Section 3 – License Conditions.
+
+Your exercise of the Licensed Rights is expressly made subject to the following conditions.
+
+    a.  Attribution.
+
+          1.    If You Share the Licensed Material (including in modified form), You must:
+                A.    retain the following if it is supplied by the Licensor with the Licensed Material:
```

Comment:
Too long lines, question reduce the font or adjust the files?

![image](https://user-images.githubusercontent.com/10553630/69626942-f0fd2380-1049-11ea-9475-743e6bd063a8.png)

Reply:
I can fix this with a smaller font in https://github.com/DANS-KNAW/easy-licenses/pull/7